### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by nassersaazi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Node api for Bitcoin Transcripts
+# Node api for BTC Transcript Review
 
-- This repo houses the api that powers the bitcoin transcripts review: [review.bitcointranscripts.com](https://review.bitcointranscripts.com)
+- This repo houses the api that powers [review.btctranscripts.com](https://review.btcranscripts.com)
```
Comment (nully0x) [inline README.md:3]: Should be This repo houses the api that powers [review.btctranscripts.com](https://review.btctranscripts.com)

Comment (nully0x) [reply README.md:3]: @nassersaazi

Comment (nassersaazi) [reply README.md:3]:
```suggestion
- Should be This repo houses the api that powers [review.btctranscripts.com](https://review.btctranscripts.com/)
```
@nully0x yes that's the update i made. the former link was wrong

Comment (nassersaazi) [inline README.md:3]:
```suggestion
- This repo houses the api that powers [review.btctranscripts.com](https://review.btctranscripts.com)
```

**Review by nully0x — APPROVED** [review]
LM

**Pull request merged**